lumen_runtime/src/scheduler.rs

```diff
@@ -123,12 +123,23 @@ impl Scheduler {
                     if !arc_process.is_exiting() {
                         match Process::run(&arc_process) {
                             Ok(()) => (),
-                            Err(exception) => unimplemented!(
-                                "{:?} {:?}\n{:?}",
-                                arc_process,
-                                exception,
-                                *arc_process.acquire_heap()
-                            ),
+                            Err(exception) => {
+                                match exception {
+                                    Exception::Alloc(_inner) => {
+                                        println!("========================================== GC ==============");
+                                        match arc_process.garbage_collect(0, &mut []) {
+                                            Ok(_freed) => (),
+                                            Err(gc_err) => panic!("Gc error: {:?}", gc_err),
+                                        }
+                                    }
+                                }
+                            }
+                            //unimplemented!(
```

Comment:
Remove the unimplemented that is commented out, but probably also remove the `println` on 129.